src/Data/Aeson/Types/FromJSON.hs

```diff
@@ -2334,6 +2334,48 @@ instance FromJSON1 Monoid.Dual where
 instance FromJSON a => FromJSON (Monoid.Dual a) where
     parseJSON = parseJSON1
 
+instance FromJSON1 Monoid.Sum where
+    liftParseJSON _ p _ a = coerce (p a)
```

Comment:
I'd expect these to look like instances for `Down` or `Dual` above.

Reply:
I used Down instances as a model instead of Min, which was the first model I used. 

I took the opportunity to uniformize all newtype instances from Data.Monoid, including Dual, Min, Max, WrappedMonoid, First and Last. I can remove my last commit if you don't like it.

Cheers !